torchci/pages/sli.tsx

```diff
@@ -130,8 +132,9 @@ export function TimeRangePicker({
           <MenuItem value={7}>Last 7 Days</MenuItem>
           <MenuItem value={14}>Last 14 Days</MenuItem>
           <MenuItem value={30}>Last Month</MenuItem>
-          <MenuItem value={90}>Last Quarter</MenuItem>
-          <MenuItem value={180}>Last Half</MenuItem>
+          <MenuItem value={60}>Last 2 Months</MenuItem>
+          <MenuItem value={90}>Last 3 Months</MenuItem>
```

Comment:
If you are changing those names, why not replace them with "Last 30 Days", "Last 60 Days" and "Last 90 Days"?

Reply:
keeping the same standard from '/cost_analysis'